app/css/components/file-editor.css

```diff
@@ -0,0 +1,5 @@
+@import "../styles";
+
+.c-file-editor {
+    height: 100%;
+}
```

Comment:
I don't think we necessarily need a whole component of this unless you think it's going to be reusable. I'd instead suggesting renaming the class to `--file-editor` and moving this CSS into the the general editor component.